Doc/library/decimal.rst

```diff
@@ -1001,8 +1001,8 @@ described below. In addition, the module provides three pre-made contexts:
    In single threaded environments, it is preferable to not use this context at
    all.  Instead, simply create contexts explicitly as described below.
 
-   The default values are :attr:`prec`\ =\ :const:`28`,
-   :attr:`rounding`\ =\ :const:`ROUND_HALF_EVEN`,
+   The default values are :attr:`!prec`\ =\ :const:`!28`,
+   :attr:`!rounding`\ =\ :const:`ROUND_HALF_EVEN`,
```

Comment:
```suggestion
   The default values are :attr:`Context.prec`\ =\ ``28``,
   :attr:`Context.rounding`\ =\ :const:`ROUND_HALF_EVEN`,
```

`prec` and `rounding` are actual attributes of `Context`/`DefaultContext` that are used in numerous examples but formally documented, so these warnings are valid.